db/createItem.go

```diff
@@ -0,0 +1,41 @@
+package db
+
+import (
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/aws/awserr"
+	"github.com/aws/aws-sdk-go/service/dynamodb"
+	"github.com/aws/aws-sdk-go/service/dynamodb/dynamodbattribute"
+	"github.com/mount-joy/thelist-lambda/data"
+)
+
+func (d *dynamoDB) CreateItem(listID *string, name *string) (*data.Item, error) {
+	itemID := d.generateID()
+
+	item := &data.Item{
+		ListID: *listID,
```

Comment:
I think `Item` will have changed by the time this is ready to merge so that these are no longer pointers